libs/filesystem/src/main/java/org/padaiyal/utilities/filesystem/wordspackage/WordCounter.java

```diff
@@ -1,4 +1,4 @@
-package org.padaiyal.mavenprojecttemplate.wordspackage;
+package org.padaiyal.utilities.filesystem.wordspackage;
```

Comment:
The words package and it's corresponding tests were template contents and need to be removed.

Reply:
good, will remove it then.